web/client-api/src/test/java/io/deephaven/web/client/api/NullValueTestGwt.java

```diff
@@ -28,12 +29,28 @@ public class NullValueTestGwt extends AbstractAsyncGwtTestCase {
                     "   \"MyString=i==0?null:``+i\",\n" +
                     "   \"MyDate=i==0?null:epochNanosToInstant(i)\",\n" +
                     "   \"MyBigInteger=i==0?null:java.math.BigInteger.valueOf(i)\",\n" +
-                    "   \"MyBigDecimal=i==0?null:java.math.BigDecimal.valueOf(i, 4)\"\n" +
+                    "   \"MyBigDecimal=i==0?null:java.math.BigDecimal.valueOf(i, 4)\",\n" +
+                    "   \"MyStringArray1=i==0?null:new String[] {`A`, `B`, `C`}\"\n" +
```

Comment:
Why `MyStringArray1` instead of just `MyStringArray`? 1 dimension?

Reply:
Yep, also to add another test for arrays that contain nulls, but given the preview issues, I didn't dig in there yet.